apps/dapp/src/components/Layouts/Ascend/utils.ts

```diff
@@ -24,12 +26,12 @@ export const createPool = (pool: SubgraphPool): Pool => {
       endTimestamp: new Date(Number(weightUpdate.endTimestamp) * 1000),
       startTimestamp: new Date(Number(weightUpdate.startTimestamp) * 1000),
       startWeights: [
-        getBigNumberFromString(weightUpdate.startWeights[0]),
-        getBigNumberFromString(weightUpdate.startWeights[1]),
+        BigNumber.from(weightUpdate.startWeights[0]),
```

Comment:
We should probably just make these DecimalBigNumbers from the start?

Reply:
if it changes too much downstream we can do a followup/cleanup after we merge this/higher priority stuff

Reply:
Yes good call, seems like an easy change will do it.

Reply:
https://github.com/TempleDAO/temple/pull/461/commits/bb507608688ba733e68f2319522ec1215bc9784f